File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/images/UpdatePsdPropertiesWithoutStorage.java
package com.aspose.imaging.cloud.examples.images;

import com.aspose.imaging.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class UpdatePsdPropertiesWithoutStorage {

    public static void main(String... args) throws IOException {
        String input = "Sample1.psd";
        String output = "Sample2.psd";
        Path inputFile = Utils.getPath(UpdatePsdPropertiesWithoutStorage.class, input);
        Path outputFile = Utils.getPath(UpdatePsdPropertiesWithoutStorage.class, output);

        Integer channelsCount = 3;
        String compressionMethod = "rle";
        Boolean fromScratch = false;
        String outPath ="";

        com.aspose.imaging.model.ResponseMessage apiResponse = Utils.getImagingSdk().PostImagePsd(
                channelsCount, 
                compressionMethod, 
                fromScratch, 
                outPath, 
                inputFile.toFile()
        );

        Files.copy(apiResponse.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/Node.js/REST/images/UpdatePsdPropertiesWithoutStorage.js
var utils = require("../utils.js");
var fs = require('fs');

var input_file = 'sample1.psd';
var inputPath = getPath(__filename, input_file);
var output_file = 'output.psd';
var outputPath = getPath(__filename, output_file);

var input_buffer = fs.readFileSync(inputPath);

var method = 'POST';
var request_url ='imaging/psd?channelsCount=3&compression=rle';

ProcessCommandStream(
    method,
    Sign(request_url),
    input_buffer,
    function(output_buffer) {
        fs.writeFileSync(outputPath, output_buffer);
    });